src/Payments.Mvc/Controllers/ReportsController.cs

```diff
@@ -144,6 +147,29 @@ public async Task<IActionResult> PendingRefundRequests()
                 .ToListAsync();
 
             return View(invoices);
+        }
+
+        [Authorize(Roles = ApplicationRoleCodes.Admin)]
+        public async Task<IActionResult> InactiveUsers()
+        {
+            var usersWithTeams = await _dbContext.TeamPermissions.Include(a => a.Team).Include(a => a.User).Include(a => a.Role).AsNoTracking().Select(TeamUsersReportModel.Projection()).ToListAsync();

```

Comment:
i don't think you need the includes, it's just extra returned data I don't see you using anywhere.